index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <main id="main">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
You should change the title from the template and give it a custom one related to your own project

Reply:
All so give your PR a significative name (what you added to the project for example)